### ISSUE TITLE
Instance of UnauthorizedError couldn't be checked with instanceof

### ISSUE THREAD

**Issue by arthurchumak** [auth0/express-jwt#292]
**Please do not report security vulnerabilities here**. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.

**Thank you in advance for helping us to improve this library!** Please read through the template below and answer all relevant questions. Your additional work here is greatly appreciated and will help us respond as quickly as possible. For general support or usage questions, use the [Auth0 Community](https://community.auth0.com/) or [Auth0 Support](https://support.auth0.com/). Finally, to avoid duplicates, please search existing Issues before submitting one here.

By submitting an Issue to this repository, you agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

Instance of UnauthorizedError couldn't be checked with instanceof

### Reproduction

package.json
```
{
  "main": "index.js",
  "type": "module",
  "dependencies": {
    "express-jwt": "^7.7.1"
  }
}

```

index.js
```
import { UnauthorizedError } from 'express-jwt';

const e = new UnauthorizedError('credentials_bad_format', new Error('a'));

console.log(e instanceof UnauthorizedError);
```

log
```
node index.js 
false
```

### Environment

> Please provide the following:

- **Version of this library used:** 7.7.1
- **Version of the platform or framework used, if applicable:** no
- **Other relevant versions (language, server software, OS, browser):** linux, node.js 16.14.2; npm 8.3.2
- **Other modules/plugins/libraries that might be involved:**

**Comment by jfromaniello** [comment]
Thank you very much for reporting this. It is fixed in `v7.7.2`

**Comment by arthurchumak** [comment]
@jfromaniello thanks you too, for such quick response!
